WordPress/Classes/Services/AccountSettingsService.swift

```diff
@@ -18,6 +18,7 @@ protocol AccountSettingsRemoteInterface {
     func getSettings(success: @escaping (AccountSettings) -> Void, failure: @escaping (Error) -> Void)
     func updateSetting(_ change: AccountSettingsChange, success: @escaping () -> Void, failure: @escaping (Error) -> Void)
     func changeUsername(to username: String, success: @escaping () -> Void, failure: @escaping () -> Void)
+    func validateUsername(to username: String, success: @escaping () -> Void, failure: @escaping (Error) -> Void)
```

Comment:
Are we still using the validate methods now we changed to the suggestion model? If not, should we remove them?